tests/braille-specs/de-g1-bidi-specs.yaml

```diff
@@ -204,9 +204,9 @@ tests:
   - ["11:25 Uhr", "⠼⠁⠁⠠⠒⠃⠑ ⠨⠥⠓⠗", {xfail: "FT: Colon should apparently be 6-25"}]
 
 # Numbers in words
-  - ["8fach", "⠼⠓⠠⠋⠁⠉⠓"]
+  - ["8fach", "⠼⠓⠠⠋⠁⠹"]
   - ["68er", "⠼⠋⠓⠠⠑⠗"]
-  - ["32stel", "⠼⠉⠃⠎⠞⠑⠇"]
+  - ["32stel", "⠼⠉⠃⠾⠑⠇", {xfail: "BT: st backtranslates as }"}]
```

Comment:
Regarding all the "BT" comments in this file: could we do something like in https://github.com/liblouis/liblouis/blob/e1e6a1885b716a491202eff2f8a154a5e4951ef2/tests/braille-specs/ur-pk-g2.yaml#L356, where Jake Kyle repeats the commented "bothDirection" tests for forward translation only?

Reply:
Yes, sure we can do that. But maybe in a different PR? Because #1100 depends on this and I'd like the two branches not to diverge too much.

Reply:
I would do it in #1100. The YAML tests need refactoring anyway, so it's a good opportunity to also do this.